app/src/main/java/io/github/droidkaigi/confsched2018/presentation/detail/SessionDetailActivity.kt

```diff
@@ -32,6 +40,8 @@ class SessionDetailActivity :
     @Inject lateinit var viewModelFactory: ViewModelProvider.Factory
     @Inject lateinit var drawerMenu: DrawerMenu
 
+    private var backPressed = false
```

Comment:
Please comment why this need 🙏

Reply:
I think it will not be needed..., I will remove this on next commit. Thanks.